src/components/PhotoAlbum/PhotoAlbum.tsx

```diff
@@ -33,10 +33,11 @@ export const PhotoAlbum = ({ photos }: { photos: any[] }) => {
           distance={Math.abs(currentPhoto - index)}
           onClick={() => setCurrentPhoto(index)}
           active={(index === currentPhoto).toString()}
-          src={`https:${photo.fields.file.url}`}
-          height={300}
-          width={600}
+          src={`https:${photo.fields.file.url}?&w=500`}
```

Comment:
```suggestion
          src={`https:${photo.fields.file.url}?&w=624&h=448`}
```

Reply:
i think this change would make the quality of the image a little bit better without affecting significantly the site's performance.

Reply:
nice suggestion!! thanks